src/Form/Type/CommentType.php

```diff
@@ -62,10 +62,7 @@ public function getName()
         return $this->getBlockPrefix();
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function setDefaultOptions(OptionsResolverInterface $resolver): void
+    public function configureOptions(OptionsResolver $resolver): void
```

Comment:
This looks strange.

Reply:
Ah, I hesitated here, let me check what the right call is

Reply:
Seems to be the correct method though, right ?
https://github.com/symfony/form/blob/master/Extension/Core/Type/BaseType.php#L110